### DISCUSSION TITLE
How to run a single solution?

### DISCUSSION THREAD

**Discussion by Google987** [PlummersSoftwareLLC/Primes#443] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/30652896/126030409-aa407eea-9e2d-4bb7-be2b-e22d2ebfe088.png)
`make one SOLUTION=PrimeCrystal/solution_1 FORMATTER=csv`
this command is suggested in the repo
https://github.com/PlummersSoftwareLLC/Primes/blob/drag-race/BENCHMARK.md#running-a-single-benchmark

But I'm getting this error. Although only `make` working fine and I'm able to run all the benchmarks at once. but I want to run a single one.

**Comment by marghidanu** [comment, marked as answer]
The documentation has changed, If you can use `make DiRECTORY=PrimesCPP` to run all solutions for a given implementation or  `make DiRECTORY=PrimesCPP/solution_1` to run just one solution. Documentation will be updated soon.

**Reply by Google987** [reply]
Thanks for such a quick reply.
`sudo make DIRECTORY=PrimeCPP/solution_1`
It worked fine. 

![image](https://user-images.githubusercontent.com/30652896/126036033-476fddbb-6e7d-4558-a330-3bd1b77cabba.png)
